tests/test_nc2mmd_script.py

```diff
@@ -130,18 +130,36 @@ def test_with_folder(dataDir, monkeypatch):
     parser = create_parser()
     in_dir = tempfile.mkdtemp()
     shutil.copy(os.path.join(dataDir, 'reference_nc.nc'), in_dir)
+    shutil.copy(os.path.join(dataDir, 'reference_nc.nc'),
+                os.path.join(in_dir, 'reference_nc_copy.nc'))
+    assert os.path.isfile(os.path.join(in_dir, 'reference_nc_copy.nc'))
```

Comment:
Why copy in the test code?
shouldn't reference_nc_copy.nc be added to git?

Reply:
No, it's not necessary to add it to git. It is just to test that an error is issued in case of repeated IDs.